app/routes/models/task-list.js

```diff
@@ -42,24 +42,27 @@ const updateSections = (agreement) => {
   const fundingSection = sections.find(x => x.name === 'Choose your funding')
   const fundingTask = fundingSection.tasks.find(x => x.name === 'Choose funding option')
 
-  // land section complete but funding not set yet
-  if (land.isLandCorrect != null && land.hasManagementControl != null && !funding.length) {
+  // land section complete
+  if (land.landComplete) {
     landSection.completed = true
     landCoverTask.status = COMPLETED
+  }
+
+  // if funding not started
+  if (!funding.length) {
     fundingSection.status = NOT_STARTED_YET
     fundingTask.status = NOT_STARTED_YET
     return sections
-  }
-
-  // funding selected, need to hide placeholder actions and determine which actions to show
-  if (funding.length) {
+  } else {
```

Comment:
Not required as you return from the function if true

Reply:
Agreed.  Will remove on next PR